pkg/cable/libreswan/libreswan.go

```diff
@@ -118,10 +135,23 @@ func (i *libreswan) refreshConnectionStatus() error {
 
 	for scanner.Scan() {
 		line := scanner.Text()
-		// Line format: 006 #3: "submariner-cable-cluster3-172-17-0-8-0-0", type=ESP, add_time=1590508783, inBytes=0, outBytes=0, id='172.17.0.8'
-		components := strings.Split(line, "\"")
-		if len(components) == 3 {
-			activeConnections.Add(components[1])
+		matches := trafficStatusRE.FindStringSubmatch(line)
+		if matches != nil {
+			activeConnections.Add(matches[1])
+
+			inBytes, err := strconv.Atoi(matches[2])
+			if err != nil {
+				klog.V(log.DEBUG).Infof("Invalid inBytes in whack output line: %q", line)
+			} else {
+				libreswanRxGauge.Set(float64(inBytes))
```

Comment:
Won't we have a different line for endpoint/subnet? we may need to make aggregations I think?

Reply:
Yes, I need to sum the values.